### PR TITLE
Aarch64 Compute

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

File: chef/cookbooks/nova/templates/default/qemu.conf.erb [hunk 1]
```diff
@@ -499,3 +516,43 @@ group = "<%= @group %>"
...
+# differently for different guest architectures. For instance, it's OVMF
+# for x86_64 and i686, but it's AAVMF for aarch64. The libvirt default
+# follows this scheme.
+#nvram = [
+#   "/usr/share/qemu/ovmf-x86_64-ms-code.bin:/usr/share/qemu/ovmf-x86_64-ms-vars.bin",
+#   "/usr/share/qemu/aavmf-aarch64-code.bin:/usr/share/qemu/aavmf-aarch64-vars.bin"
+#]
+
+nvram = [
+   "/usr/share/qemu/ovmf-x86_64-ms-code.bin:/usr/share/qemu/ovmf-x86_64-ms-vars.bin",
```
Comment (vuntz) [inline chef/cookbooks/nova/templates/default/qemu.conf.erb]: AFAIK, we don't install any package for this one so we don't have it.

Comment (dirkmueller) [inline chef/cookbooks/nova/templates/default/qemu.conf.erb]: Yeah, but thats a bug not a feature. we can install it though (it is part of SLES). do you want me to handle that in this PR?

Comment (vuntz) [inline chef/cookbooks/nova/templates/default/qemu.conf.erb]: Ok, I'm fine leaving it then. And feel free to install the package for x86_64 too.

Comment (dirkmueller) [inline chef/cookbooks/nova/templates/default/qemu.conf.erb]: added a commit for that

File: chef/cookbooks/tempest/recipes/config.rb [hunk 2]
```diff
@@ -426,7 +426,7 @@
   use_interface_attach = true
   use_rescue = true
   use_suspend = true
-  use_vnc = true
+  use_vnc = node[:kernel][:machine] != "aarch64"
```
Comment (vuntz) [inline chef/cookbooks/tempest/recipes/config.rb]: Not sure it really matters, but I guess we should check the architecture of one of the compute node.

**Comment by vuntz** [comment]
+1

File: chef/cookbooks/tempest/recipes/config.rb (hunk 2 above)
Comment (dirkmueller) [inline chef/cookbooks/tempest/recipes/config.rb]: I was thinking about that, but decided against it. I think the implicit assumption is that we use tempest against the architecture environment where tempest is being deployed on. there is only one image to reference in the tempest.conf so running multi-architecture tempest doesn't work anyway..

This way you can choose to test aarch64 or x86_64 in a dual-architecture environment based on where you deploy tempest on. (yes, I know tempest doesn't handle aarch64 yet).

**Comment by jsuchome** [comment]
+1. I do not know about Aarch64, but the code makes sense to me.

**Pull request merged**